yarn.lock

```diff
@@ -4309,7 +4309,7 @@ __metadata:
     garbo-lib: ^0.0.1
     grimoire-kolmafia: ^0.3.23
     kolmafia: ^5.27628.0
-    libram: ^0.8.13
+    libram: latest
```

Comment:
If you need to bump the version then it should specify the version number. There are now two versions in this lock file due to the way it was added.

Reply:
Fixed!